docs/ldml/tr35.md

```diff
@@ -1299,6 +1299,9 @@ While text in a document can be tagged as partly in one language and partly in a
 
 To communicate requests for localized content and internationalization services, locales are used. When people pick a language from a menu, internally they are picking a locale (en-GB, es-419, etc.). To allow an application to support Spanglish or Hinglish locale selection, [unicode_locale_id](#unicode_locale_id)s can represent hybrid locales using the T extension key-value 'h0-hybrid'. (For more information on the T extension, see _Section 3.7 [Unicode BCP 47 T Extension](#t_Extension)._)
 
+_However, if users typically expect a their language in non-default script to contain an admixture of another language, then the -t- and hybrid subtags may be omitted.
+Thus if Hindi written in Latin script normally contains an admixture of English text, ‘hi-Latn’ can be used instead of ‘hi-Latn-t-en-h0-hybrid’._
```

Comment:
consider `a significant amount of English text due to lexical borrowing` instead of `admixture of English text`